scripts/HRRR/download_hrrr.sh

```diff
@@ -86,100 +95,109 @@ check_file_existence(){
   fi
   return 1
 }
+export -f check_file_existence
+
+download_hrrr() {
+  DAY_HOUR=$1
+  FC_HOUR=$2
+  FILE_NAME="hrrr.t$(printf "%02d" $DAY_HOUR)z.wrfsfcf0${FC_HOUR}.grib2"
+
+  printf "  File: ${FILE_NAME}"
+
+  check_file_existence
+  if [[ $? -eq 0 ]]; then
+    return
+  elif [[ -e ${FILE_NAME} ]]; then
+    rm ${FILE_NAME}
+  fi
```

Comment:
Added this step to remove a previously downloaded file that was 0 in size to download it again.